### ISSUE TITLE
Changes to SOCKS and AWS access requests

### ISSUE THREAD

**Issue by rmtolmach** [department-of-veterans-affairs/va.gov-team#38719]
### Description
I'm making some changes to the workflow for requesting SOCKS and AWS access (going forward, the requests will be made on two separate forms). I will need some edits, mostly in the "Request access" section of the "Request access to tools" page. How can I go about that? I can submit new text or relevant information. let me know. thanks!

### Relevant URLs
This page: https://depo-platform-documentation.scrollhelp.site/getting-started/Request-access-to-tools.969605215.html#Requestaccesstotools-Additionalaccessfordevelopers

The ticket I'm working on is https://github.com/department-of-veterans-affairs/va.gov-team/issues/31272

**Comment by gmaciolek** [comment]
Hi, @rmtolmach! Thanks for reaching out. The [Platform website guidelines](https://vfs.atlassian.net/l/c/Gq4kUXYT) explain how to add and edit content on the Platform website. For the changes you're describing, you'll specifically need the instructions to [Edit a Platform website document](https://vfs.atlassian.net/l/c/8bkQGSWD), but I recommend starting on the main Platform website guidelines page to familiarize yourself with the general guidelines.

Editing documents (vs. adding new documents) is a cumbersome process. If you get lost in the instructions, our team is happy to help. You can ask a question in #vsp-confluence or attend our office hours on Tuesday or Thursday mornings at 11AM ET. The Zoom link and meeting details are posted in #vsp-confluence.

**Comment by rmtolmach** [comment]
Alright... I think I'm good @gmaciolek. Thanks for your help! I've added my changes to Confluence https://vfs.atlassian.net/wiki/spaces/GS/pages/969605215/Request+access+to+tools and I'll see them go live on Tuesday! And I deleted my drafts from the Platform website draft space ✅ . 

The only problem I had was when I pasted my draft page into the official Confluence space, it deleted comments that had been made in areas that I didn't actually change. (I only changed content in the bottom half of the page, but some of the comments were in the top of the page.) They can still be seen as "Resolved comments". I wasn't quite sure what action needed to be taken on the comments. Just wanted to make you aware!
